mkdocs.yml

```
@@ -13,6 +13,8 @@ nav:
     - 'recheck/how-ignore-works-in-recheck.md'
     - 'recheck/location-of-the-golden-master-files.md'
     - 'recheck/reuse-golden-master-files.md'
+  - recheck-cli:
+    - 'recheck-cli/setup.md'
```

This typo (`-` instead of `.` in `recheck-cli`) probably lets the build fail:

```suggestion
    - 'recheck.cli/setup.md'
```